pkg/chartmuseum/server/multitenant/server.go

```diff
@@ -157,8 +159,17 @@ func NewMultiTenantServer(options MultiTenantServerOptions) (*MultiTenantServer,
 		TenantCacheKeyLock:     &sync.Mutex{},
 		CacheInterval:          options.CacheInterval,
 		ChartLimits:            l,
+		WebTemplatePath:        options.WebTemplatePath,
 	}
 
+	// add static route and load html templates
+	if server.WebTemplatePath != "" {
```

Comment:
Unfortunately, by adding the routes here, these pages are no longer protected by auth.

All incoming routes are currently defined in this [file](https://github.com/helm/chartmuseum/blob/main/pkg/chartmuseum/server/multitenant/routes.go) and auth is handled [here](https://github.com/helm/chartmuseum/blob/main/pkg/chartmuseum/router/router.go#L212).

I think we'll have to take a different approach, though I'm not sure the best way forward off the top of my head! 🤔

Reply:
Good point about the auth part! I forgot about this. As we use allow the get for anonymous!

Do you have any suggestions? I am happy to do the rework.

Reply:
@dirien I'll have to dig in a bit more but I'm fairly certain we can move the Auth code to be a middleware so it applies to all routes.

Reply:
Sounds good! I will check this too, 2morrow! thanks for the hint! much appreciate it!

Reply:
@cbuto, moved the delivery of potential static files into a handler, behind the auth. 
Tests looked good so far.

Reply:
nice! at first glance, looks good!